### ISSUE TITLE
Document Accessing with Podman a network behind a user-controlled VPN on Windows

### ISSUE THREAD

**Issue by themr0c** [containers/podman-desktop#3569]
Document Accessing with Podman a network behind a user-controlled VPN on Windows

TLDR: In Podman Desktop 1.3.0 or greater, when initializing the Podman machine, select the **User mode networking** option. See: https://github.com/containers/podman-desktop/pull/3251, https://github.com/containers/podman-desktop/issues/2865 https://github.com/containers/podman-desktop/issues/3556 

Procedure initial draft:

# Accessing with Podman a network behind a user-controlled VPN on Windows

On Windows, when Podman requires to access resources that are behind a user-controlled VPN, enable the user mode networking in your Podman machine.

#### Prerequisites

- Windows host with updated WSL2.
- Podman 4.6.0 or greater.
- On the host, the user controls the VPN activation.

#### Procedure

1. To enable user mode networking, when creating the Podman machine, select the **User mode networking (traffic relayed by a user process).** option. 

2. SSH to the Podman machine

   ```
   $ podman machine ssh *<podman_machine_name>*
   ```

3. Create and edit the `/etc/resolv.conf/` file:

   ```
   nameserver 1.1.1.1
   nameserver 8.8.8.8
   ```

4. Create and edit the `/mnt/wsl/wsl.conf` file:
   
   ```
   [network]
   generateResolvConf = false
   ```

5. Exit the Podman machine SSH session.

   ```shell-session
   $ exit
   ```

6. Restart the Podman machine.

#### Verification

- You can access resources that are behind the VPN.

_Originally posted by @themr0c in https://github.com/containers/podman-desktop/discussions/3151#discussioncomment-6748041_

**Comment by themr0c** [comment]
This week and next week I don't have access to my Windows laptop. Therefore I am writing blindly, what I *assume* to be right, without the opportunity to test and assert. I would be grateful if someone can test it for me. I find it weird that we have to edit `resolv.conf` and `wsl.conf` manually.

**Comment by anil-kk** [comment]
This actually works, tested!
